link AWSDNSRecord and LoadBalancerV2

serain opened the pull request:
This resolves issue #488 

Links AWSDNSRecord to LoadBalancerV2.

achantavy reviewed (changes requested):
Thank you for working on this!

Can you please also update the tests?

For example, this code tests if ELBv2 nodes are attached to EC2 instances:
https://github.com/lyft/cartography/blob/master/tests/integration/cartography/intel/aws/ec2/test_ec2_load_balancers.py

and the source sample test data is here:
https://github.com/lyft/cartography/blob/master/tests/data/aws/ec2/load_balancers.py

Let us know if you need help on this.

serain commented:
I gave this a crack, but I'm not sure it's what you had in mind.

I added a test to route53 (felt appropriate as it's the route53 module that creates the links). I added a DNS record to the test data and then load both the EC2 and Route53 test data and check there's the expected link.

Let me know what you think.

achantavy reviewed (approved):
This looks great, thanks @serain!
